build/secrets/.secrets.baseline

```diff
@@ -616,7 +616,7 @@
         "filename": "build/lib/stylelint/vscode-known-variables.json",
         "hashed_secret": "bebac042cf8ae1cf5954a7f233759a1373a1bd5b",
         "is_verified": false,
-        "line_number": 775,
+        "line_number": 771,
```

Comment:
I was surprised to see this line number change 🤔 Was updating the secrets missed in another PR to this branch?

Reply:
Hm, it must've been. I'm not sure but I just reran the `update-baseline` script and it didn't modify it out so I'm going to leave it.